code/modules/overmap/exoplanets/decor/_turfs.dm

```diff
@@ -34,17 +35,15 @@
 			if(exoplanet.planetary_area && istype(loc, world.area))
 				ChangeArea(src, exoplanet.planetary_area)
 		// if away site
-		else if(istype(site, /datum/map_template/ruin/away_site))
-			var/datum/map_template/ruin/away_site/away_site = site
+		else if(istype(template, /datum/map_template/ruin/away_site))
+			var/datum/map_template/ruin/away_site/away_site = template
 			if(away_site.exoplanet_atmosphere)
 				initial_gas = away_site.exoplanet_atmosphere.gas.Copy()
 				temperature = away_site.exoplanet_atmosphere.temperature
-			else
-				initial_gas = list()
-				temperature = T0C
-			set_light(MINIMUM_USEFUL_LIGHT_RANGE, away_site.lightlevel, away_site.lightcolor)
+			if(away_site.exoplanet_lightlevel && is_outside())
+				set_light(MINIMUM_USEFUL_LIGHT_RANGE, away_site.exoplanet_lightlevel, away_site.exoplanet_lightcolor)
 
-	// if not on an exoplanet, instead just keep the default or mapped in atmos
+	// if not on a exoplanet, instead just keep the default or mapped in atmos
```

Comment:
The grammar was correct before, e is a vowel so the junction is `an`

Reply:
changed back to 'an'